module/VuFind/src/VuFind/Controller/AlmaController.php

```diff
@@ -348,58 +349,49 @@ protected function webhookChallenge()
      * Send the "set password email" to a new user that was created in Alma and sent
      * to VuFind via webhook.
      *
-     * @param UserEntityInterface    $user   A user row object from the VuFind
-     * user table.
+     * @param UserEntityInterface    $user   User entity object
      * @param \Laminas\Config\Config $config A config object of config.ini
      *
      * @return void
      */
-    protected function sendSetPasswordEmail($user, $config)
+    protected function sendSetPasswordEmail(UserEntityInterface $user, $config)
     {
-        // If we can't find a user
-        if (null == $user) {
```

Comment:
There was no code path that would allow a null $user value to be passed here, so I removed the unnecessary checking and added a strict type to prevent inappropriate future changes.